### PR TITLE
Implementing search for cloudflow.io docs

### PR DESCRIPTION
**Pull request by andreaTP**
### What changes were proposed in this pull request?
Implement a "search" box for the cloudflow.io site

### Why are the changes needed?
Be able to search for content in the documentation site

### Does this PR introduce any user-facing change?
Improvement in the documentation

### How was this patch tested?
```bash
cd docs
make html-author-mode
```

_NOTE:_
This PR is still a Draft since it needs some CSS love, the search is already working but looks pretty ugly ATM, I hope someone like @rstento and/or @rasummer can help here 🙏  thanks!

### CODE REVIEW

File: docs/antora-docker/README.md [hunk 1]
```diff
@@ -1,17 +1,15 @@
-# Docker image containing Antora and the tools used to generate the doc
+Docker image containing Antora and the tools used to generate the doc
 
 ## Build the image
 
 ```
-docker build -t lighbend/antora-doc:<version> .
+docker build . -t lightbend/antora-cloudflow-doc:<version>
```
Comment (RayRoestenburg) [inline docs/antora-docker/README.md:6]: 👍

**Review by RayRoestenburg — APPROVED** [review]
ugly search is better than no search! ;-)

**Comment by rstento** [comment]
@andreaTP, I was able to get the styling from Richard's css to apply by:

1) Adding a heading-styles.hbs file to supplementatal-ui/partials, with this content:
<link rel="stylesheet" href="{{uiRootPath}}/css/theme.1.0.2.min.css">

2) Copying head.hbs from https://github.com/lightbend/antora-ui-lightbend-cloud-theme/blob/master/src/partials/head.hbs to supplemenatal-ui/partials and adding this line: 
{{> head-styles}}

However, there is a clash between the UI in https://github.com/lightbend/antora-ui-lightbend-cloud-theme/blob/master/src/partials/head.hbs and these new styles for the search box and the search results do not display correctly.

I know @rasummer  plans to expand the https://lightbend.github.io/antora-supplemental-ui-lightbend-theme/ to provide a theme for the open source docs, Cloudflow and Cloudstate, and it looks to me like he is partly done. It might be better to wait until he is and then we can switch over. He's on vacation today so I expect he'll weigh in when he gets back.

**Comment by rasummer** [comment]
I'm mostly done getting the new theme (https://lightbend.github.io/antora-supplemental-ui-lightbend-theme/build/site/index.html) ready for open source docs. It looks like we might be mixing the old theme, with some elements from the new theme here. I might be able to take a deeper look next week.

**Comment by RayRoestenburg** [comment]
Thanks @rasummer we're really looking forward to search in the OSS version!

**Pull request merged**